animad-service-administration/animad-administration-service/src/main/java/de/muenchen/animad/admin/administration/service/gen/rest/search/Keeper_SearchController.java

```diff
@@ -42,6 +43,9 @@
     @Autowired
     Keeper_Repository repository;
 
+    @Value("${service.configuration.maxSearchResults}")
```

Comment:
Ich würde hier evtl. einen Standardwert vergeben, da die Anwendung, wenn der Wert in der Konfigruation nicht vorhanden ist, nicht hochfährt.